.yamllint

```diff
@@ -0,0 +1,10 @@
+---
+
+extends: default
+
+rules:
+  indentation:
+    indent-sequences: consistent
+  comments-indentation: disable
+  document-start: disable
+  line-length: disable
```

Comment:
`yamllint` seems to be stuck on YAML 1.1 definitions of truthy values, so complains that `on` should not be used because it is a boolean: `truthy value should be one of [false, true]`.

This hasn't been true since YAML spec 1.2 (released in 2009), where only `true`, `True`, `TRUE`, `false`, `False`, and `FALSE` are treated as boolean unless quoted.

Our `release.yml` file uses an unquoted `on` in 2 places:

```yaml
on:
  push:
    tags:
      - 'v*'
      - 'test-action-release-*'
env:
  GO111MODULE: on
```

We can tell `yamllint` to ignore it via:

```yaml
rules:
  truthy:
    allowed-values: ['true', 'false', 'on']
```

This is kind of bogus for YAML 1.2, but does turn off the warnings, so maybe it is good enough for us.